src/implementations/vanilla/components/gallery/gallery.js

```diff
@@ -354,6 +355,37 @@ export class Gallery {
     }
   }
 
+  iframeResize(iframe) {
+    if (!iframe && this.overlay) {
+      iframe = queryOne('iframe', this.overlay);
+    }
+
+    if (iframe) {
+      const width = window.innerWidth;
+      const height =
+        this.overlay.clientHeight -
+        this.overlayHeader.clientHeight -
+        this.overlayFooter.clientHeight;
+
+      if (width > height) {
+        iframe.setAttribute('height', `${height}px`);
+
+        if ((height * 16) / 9 > width) {
```

Comment:
I don't really understand why we are still using predefined ratio here (16/9 and 4/3).
If I'm not wrong, we have the available height and width, but why assuming that the video is using one ratio or the other?

Reply:
yes, i was fixing that while you reviewed this, should be ok now. 
About the ratio, before we were always forcing 16/9, the idea here is to keep that format as long as it possible without cropping the video in landscape viewports and switching to 4/3 in portrait.
Like this we can mantain a certain consistency also about the ratios, but yes we could also forget about it and always try to fit the container..